tests/Specs/Pack.cs

```diff
@@ -367,6 +367,7 @@ public void Pack_App()
                 List<string> appEntries = TestUtilities.GetFileEntriesFromZip(packageZipPath);
                 Assert.True(appEntries.HasAny(entry => entry == appManifest), "App manifest file does not exist");
                 Assert.True(appEntries.HasAny(entry => entry == "app_icon.png"), "App Icon does not exist");
+                Assert.True(appEntries.HasAny(entry => entry == "app_deployment_manifest.xml"), "Deployment manifest does not exist");

```

Comment:
I thought the deployment manifest was supposed to be loaded as metadata to the store, and not included in the App manifest package?